### ISSUE TITLE
JMavSim not launching with fresh copy from master

### ISSUE THREAD

**Issue by crashmatt** [PX4/Firmware#6967]
Fresh copy of repo:
https://github.com/PX4/Firmware.git

Do
make posix_sitl_default jmavsim

Results in 
INFO  [simulator] Waiting for initial data on UDP port 14560. Please start the flight simulator to proceed..

Diff with old working copy shows that JMavSim has moved or has gone.  Many other libraries appear to be missing or not built.

**Comment by bkueng** [comment]
```
git clone --recursive https://github.com/PX4/Firmware.git
cd Firmware
make posix_sitl_default jmavsim
```
works fine for me. Did you clone the submodules?

**Comment by crashmatt** [comment]
@bkueng I normally use git gui and recursive clone option.
I tried your cmd line version but it does the same

Now I found the error:
/home/matt/src/Firmware/Tools/jmavsim_run.sh: line 44: ant: command not found

Ant is not available on my machine might answer why it can't build jmavsim.  The older repo clone already has jmavsim built so it doesn't have this problem.
My linux machine has gone through a few updates recently.  Ant may have disappeared.  I will investigate and probably close.
